rclrs/minimal_pub_sub/Cargo.toml

```diff
@@ -30,6 +30,7 @@ anyhow = {version = "1", features = ["backtrace"]}
 rclrs = "0.6"
 rosidl_runtime_rs = "0.5"
 example_interfaces = "*"
+clap = { version = "4.5.51", features = ["derive"] }
```

Comment:
Usually dependencies are only so specific if they need a fix that is present in the requested minor / patch version. Does your example need anything specific from `4.5.51` or is `4.5`, or even `4` good enough? If so can you just change it to the less strict requirement?